stylesheets/style.css

```diff
@@ -12,3 +12,15 @@ footer {
   min-width: 680px;
   margin: 0 auto;
 }
+
+.tutorial-filelist__list {
+}
+
+.hidden {
+	display: none;
+	visibility: hidden;
```

Comment:
`display:none` is not very accessible. I personally prefer [the more tested way used in the HTML5 boilerplate](https://github.com/h5bp/html5-boilerplate/blob/5.3.0/dist/css/main.css#L126) to hide elements visually.